### PR TITLE
Backend routes

### PR DESCRIPTION
**Pull request by arpit1912**
the pr cover the following:
1. Models for Comment and Post
2. Two donation routes

### CODE REVIEW

File: models/Comment.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+            type:String,
+            required:true
+        },
+        datetime:{
+            type:String
+        }
+    }],
+})
+
+commentSchema.index({ location: "2dsphere" })
```
Comment (krish7777) [inline models/Comment.js]: no need of this

File: models/Post.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        required: true
+    }],
+    commentId:{
+        type: Schema.Types.ObjectId,
+        ref: 'Comment',
+        required: true
+    }
+})
+
+postSchema.index({ location: "2dsphere" })
```
Comment (krish7777) [inline models/Post.js:40]: here also

**Review by krish7777 — APPROVED** [review]

File: models/Comment.js (hunk 1 above)
Comment (arpit1912) [reply models/Comment.js]: Got it!

File: models/Post.js (hunk 2 above)
Comment (arpit1912) [reply models/Post.js:40]: Ok

**Pull request merged**